upstream/kueue/e2e-test-ocp.sh

```
@@ -84,9 +87,32 @@ label_worker_nodes
+}
+
+function restore_kueue_for_folder() {
+  local folder="$1"
+  case "$folder" in
+    customconfigs)
+      echo "Removing admissionFairSharing from cluster Kueue CR..."
+      $OC patch kueue.kueue.openshift.io/cluster --type=json -p \
+        '[{"op":"remove","path":"/spec/config/admissionFairSharing"}]'
+      ;;
+  esac
+}
```

Shellcheck (0.11.0)

[info] This function is never invoked. Check usage (or ignored if invoked indirectly).

(SC2329)